[2023-06-09T14:48:27.793+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2023-06-08T00:00:00+00:00 [queued]>
[2023-06-09T14:48:27.799+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2023-06-08T00:00:00+00:00 [queued]>
[2023-06-09T14:48:27.799+0000] {taskinstance.py:1308} INFO - Starting attempt 2 of 7
[2023-06-09T14:48:27.807+0000] {taskinstance.py:1327} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2023-06-08 00:00:00+00:00
[2023-06-09T14:48:27.811+0000] {standard_task_runner.py:57} INFO - Started process 1449 to run task
[2023-06-09T14:48:27.815+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'insert_into_table', 'scheduled__2023-06-08T00:00:00+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpiy9im_pq']
[2023-06-09T14:48:27.817+0000] {standard_task_runner.py:85} INFO - Job 26: Subtask insert_into_table
[2023-06-09T14:48:27.864+0000] {task_command.py:410} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2023-06-08T00:00:00+00:00 [running]> on host 94560a00d375
[2023-06-09T14:48:27.933+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='znawawi' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v03' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2023-06-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-06-08T00:00:00+00:00'
[2023-06-09T14:48:27.934+0000] {sql.py:262} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2023-06-08', 'dag_with_postgres_operator_v03')
        
[2023-06-09T14:48:27.942+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-09T14:48:27.990+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-09T14:48:28.002+0000] {sql.py:375} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2023-06-08', 'dag_with_postgres_operator_v03')
        , parameters: None
[2023-06-09T14:48:28.005+0000] {sql.py:384} INFO - Rows affected: 1
[2023-06-09T14:48:28.014+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=insert_into_table, execution_date=20230608T000000, start_date=20230609T144827, end_date=20230609T144828
[2023-06-09T14:48:28.068+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-09T14:48:28.088+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
